xarray/backends/plugins.py

```diff
@@ -55,8 +54,11 @@ def backends_dict_from_pkg(pkg_entrypoints):
     backend_entrypoints = {}
     for pkg_ep in pkg_entrypoints:
         name = pkg_ep.name
-        backend = pkg_ep.load()
-        backend_entrypoints[name] = backend
+        try:
+            backend = pkg_ep.load()
+            backend_entrypoints[name] = backend
+        except Exception as ex:
+            warnings.warn(f"Engine {name} loading failed:\n{ex}", RuntimeWarning)
```

Comment:
Minor.
```suggestion
            warnings.warn(f"Engine {name!r} loading failed:\n{ex}", RuntimeWarning)
```
This adds `"` around the string.